docs/Tutorials/Quorum-Dev-Quickstart.md

```diff
@@ -29,4 +32,4 @@ npx quorum-dev-quickstart
 Follow the prompts displayed.
 
 When installation is complete, refer to `README.md` in the installation directory for more information
-on your test network.
+on your test network. Optionally refer to the walkthrough above
```

Comment:
"Optionally, refer to the [guide/walk-through] previously mentioned."

Reply:
done